pkg/collector/corechecks/cluster/ksm/kubernetes_state_aggregators.go

```diff
@@ -106,6 +138,52 @@ func (a *countObjectsAggregator) accumulate(metric ksmstore.DDMetric) {
 	a.accumulator[labelValues]++
 }
 
+func (a *lastCronJobCompleteAggregator) accumulate(metric ksmstore.DDMetric) {
+	a.aggregator.accumulate(metric, metrics.ServiceCheckOK)
+}
+
+func (a *lastCronJobFailedAggregator) accumulate(metric ksmstore.DDMetric) {
+	a.aggregator.accumulate(metric, metrics.ServiceCheckCritical)
+}
+
+func (a *lastCronJobAggregator) accumulate(metric ksmstore.DDMetric, state metrics.ServiceCheckStatus) {
+	if condition, found := metric.Labels["condition"]; !found || condition != "true" {
+		return
+	}
+	if metric.Val != 1 {
+		return
+	}
+
+	namespace, found := metric.Labels["namespace"]
+	if !found {
+		return
+	}
+
+	jobName, found := metric.Labels["job_name"]
+	if !found {
+		return
+	}
+
+	dashPosition := strings.LastIndexByte(jobName, '-')
+	if dashPosition == -1 {
+		return
+	}
+
+	cronjobName := jobName[:dashPosition]
+	cronjobID := jobName[dashPosition+1:]
+	id, err := strconv.Atoi(cronjobID)
+	if err != nil {
```

Comment:
a debug log here can be useful

Reply:
Debug log added: https://github.com/DataDog/datadog-agent/pull/11139/commits/d1749e22248468ca55648a9f8188928069d99492#diff-6a3196968349f588781cd7ed6e77e5136bbf45fec44ba95e257eb027de06c4aeR167